src/server/saved_objects/client/saved_objects_client.js

```diff
@@ -12,8 +12,23 @@ export class SavedObjectsClient {
     this._callAdminCluster = callAdminCluster;
   }
 
-  async create(type, body = {}) {
-    const response = await this._withKibanaIndex('index', { type, body });
+  /**
+   * Persists an object
+   *
+   * @param {string} type
+   * @param {object} body - { attributes: {}, id: myId }
+   * @param {object} options
+   * @param {boolean} options.overwrite - defaults to false
+   * @returns {promise}
+  */
+  async create(type, body = {}, options = {}) {
```

Comment:
Hmm, good question... I'd just making it optional. We won't ever be able to remove it